<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDlg</class>
 <widget class="QDialog" name="OptionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabNormal">
      <attribute name="title">
       <string>常规</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>691</width>
         <height>321</height>
        </rect>
       </property>
       <property name="title">
        <string>默认选项</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="comboEncoders">
            <item>
             <property name="text">
              <string>hevc_nvenc</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCheckEncoders">
            <property name="text">
             <string>检测编码器</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboDecoders"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnCheckDecoders">
            <property name="text">
             <string>检测解码器</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEditOutputDir"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnOutputDir">
            <property name="text">
             <string>设置输出路径</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="labelWidth">
            <property name="text">
             <string>宽</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditWidth">
            <property name="text">
             <string>1280</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelHeight">
            <property name="text">
             <string>高</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditHeight">
            <property name="text">
             <string>720</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetResolution">
            <property name="text">
             <string>选择分辨率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditBitrate">
            <property name="text">
             <string>1M</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBitrate">
            <property name="text">
             <string>选择码率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditQuality">
            <property name="text">
             <string>28</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnConvertQuality">
            <property name="text">
             <string>转码质量</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>转码速度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboPreset">
            <item>
             <property name="text">
              <string>快</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>正常</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>慢</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboHwAccelers">
            <item>
             <property name="text">
              <string>cuda</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCheckHwAccelers">
            <property name="text">
             <string>检测硬件加速</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelHwaccel">
            <property name="text">
             <string>硬件加速说明：</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="geometry">
        <rect>
         <x>-30</x>
         <y>600</y>
         <width>800</width>
         <height>29</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabEncode">
      <attribute name="title">
       <string>编码</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
